servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -121,7 +121,8 @@ def get_job(id, **kwargs):
     failures = [
         format_task_failure(name, m)
         for name, metadata in job.get('calls', {}).items() for m in metadata
-        if m.get('failures') is not None
+        if m.get('failures') is not None and (
```

Comment:
This feels like a weird way to split the line but I'm not a pythonista anymore so 🤷‍♂

Reply:
yapf made me do it